distributions/common/k8s/README.md

```diff
@@ -3,6 +3,33 @@
 This directory contains configuration for Data Transfer Project to run
 with Kubernetes using the Docker container format.
 
+## First-time kubectl setup 
+
+Kubernetes' command line utility, kubectl, must be authenticated and configured to point to your cluster(s).
+
+### Setup for an existing GKE (Google Container Engine) project
```

Comment:
Should we have a section for the Azure section?

Reply:
Added